liblangutil/Token.cpp

```diff
@@ -126,22 +126,6 @@ int precedence(Token tok)
 }
 #undef T
 
-namespace
-{
-int parseSize(string::const_iterator _begin, string::const_iterator _end)
-{
-	try
-	{
-		int m = boost::lexical_cast<int>(boost::make_iterator_range(_begin, _end));
```

Comment:
To check for zeros at the beginning, can't we just compare the two strings `to_string(m)` and `boost::make_iterator_range(_begin, _end)`?

Reply:
Sure but we're also trying to get rid of boost, don't we?

Reply:
No idea about removing boost.

Reply:
See #7259.

Reply:
It'd have been nice, if we could have replaced it with https://en.cppreference.com/w/cpp/utility/from_chars, though - but unfortunately that works with raw pointers, not iterators... given that not worth it...